src/forestclaw2d.c

```diff
@@ -1863,38 +1859,40 @@ indirect_match (int *pi,
     FCLAW_ASSERT (rpatchno != NULL && rfaceno != NULL);
 
     *rproc = pi;
-    *rblockno = pi + 2;
-    *rpatchno = pi + 3;
-    *rfaceno = pi + 5;
+    *rblockno = pi + P4EST_HALF;
+    *rpatchno = pi + P4EST_HALF + 1;
+    *rfaceno = pi + P4EST_CHILDREN + 1; /* P4EST_CHILDREN == 2 * P4EST_HALF */
```

Comment:
please can you use P4EST_CHILDREN only if referring to the volumetric refinement? In this code, it seems more to the point to write `pi + P4EST_HALF + 1 + P4EST_HALF`.